corehq/ex-submodules/casexml/apps/case/tests/test_from_xform.py

```diff
@@ -9,7 +9,7 @@
 from corehq.form_processor.interfaces.processor import FormProcessorInterface
 from corehq.form_processor.models import CaseTransaction, CommCareCaseSQL
 from corehq.form_processor.tests.utils import run_with_all_backends
-
+from corehq.form_processor.backends.couch.update_strategy import _coerce_to_datetime
```

Comment:
it might be worth moving `_coerce_to_datetime` and removing the leading `_` if it's going to be used as a utility in tests